### PR TITLE
search process

### PR DESCRIPTION
**Pull request by harethriyadh**

### CODE REVIEW

File: src/controllers/searchController.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const Dish = require('../models/dish'); // Replace with the correct path to your Dish model
```
Comment (motaz99) [inline src/controllers/searchController.js:1]: We don't need this comment, we better delete it.

**Review by ozlemkeles — APPROVED** [review]
good job Harith 👍

File: src/controllers/searchController.js
Comment (motaz99) [inline src/controllers/searchController.js:1]: I think we don't need a separate file for this search controller, we should put with the client's controllers as soon as it is related to the client right?

File: src/routes/clientRoutes.js [hunk 2]
```diff
@@ -39,4 +44,6 @@ router.get(
 
 router.get('/chefs/:id', isUserCreatedClient, clientController.getChefById);
 
+router.post('/search', isUserCreatedClient, searchController.searchDish);
+
```
Comment (motaz99) [inline src/routes/clientRoutes.js]: We are getting dishes when we search for a dish right? so This should be a get request not post request.

**Review by motaz99 — CHANGES_REQUESTED** [review]

**Review by motaz99 — APPROVED** [review]
Great work mate!

**Pull request merged**